frontend/pages/hosts/ManageHostsPage/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const ALL_HOSTS_LABEL = "all-hosts";
+export const ALL_HOSTS_LABEL = "";
```

Comment:
an empty string as an exported constant is a bit confusing, would it be possible to just use `""` where the constant is used or add a comment here?